app/models/order_balance.rb

```diff
@@ -21,6 +21,10 @@ def to_f
     end
   end
 
+  def to_html
+    "#{label}:<strong>#{amount}</strong>".html_safe
```

Comment:
I feel like wrapping the output in `<strong>` tags to make it **bold** shouldn't be a decision that's made in the model, it should stay in the view layer, right?

Reply:
Yes, although I thought it wasn't worth extracting something else. In the end, it's just as convenient as with `Money`'s #to_html